manifests/init.pp

```diff
@@ -52,6 +52,7 @@
   $arch                  = $::architecture,
   $collection            = 'PC1',
   $is_pe                 = $::puppet_agent::params::_is_pe,
+  $manage_pki_dir        = true,
```

Comment:
Can you add a comment above describing this option?

Reply:
Done.